source/fe/fe_values.cc

```diff
@@ -2191,9 +2162,9 @@ namespace FEValuesViews
     Assert(fe_values->update_flags & update_hessians,
            (typename FEValuesBase<dim, spacedim>::ExcAccessToUninitializedField(
              "update_hessians")));
-    Assert(laplacians.size() == fe_values->n_quadrature_points,
+    Assert(laplacians.size() == fe_values->n_quadrature_points /*TODO ?*/,
```

Comment:
Any plans on this todo and the ones below it? I assume this is because we resize the arrays to `max_n_quadrature_points`?

Reply:
I also think it was related to `max_n_quadrature_points` but I don't think I had any issues till now. So I remove these TODOs.